graylog2-web-interface/src/components/content-packs/ContentPackVersions.jsx

```diff
@@ -51,9 +100,17 @@ class ContentPackVersions extends React.Component {
         </td>
         <td>{pack.rev}</td>
         <td className="text-right">
-          <Button bsStyle="primary" bsSize="small" onClick={() => { this.props.onDeletePack(pack.id, pack.rev); }}>Delete</Button>
-          &nbsp;
-          <Button bsStyle="info" bsSize="small" onClick={() => { downloadRef.open(); }}>Download</Button>
+          <ButtonToolbar className="pull-right">
+            <Button bsStyle="success" bsSize="small" onClick={() => { downloadRef.open(); }}>Download</Button>
+            <DropdownButton id={`action-${pack.rev}`} bsStyle="info" title="Actions" bsSize="small">
+              <LinkContainer to={Routes.SYSTEM.CONTENTPACKS.edit(encodeURIComponent(pack.id), encodeURIComponent(pack.rev))}>
+                <MenuItem >Edit clone</MenuItem>
+              </LinkContainer>
+              <MenuItem onClick={() => { this.props.onDeletePack(pack.id, pack.rev); }}>Delete</MenuItem>
+              <MenuItem onClick={openFunc}>Install</MenuItem>
```

Comment:
I know it's a small thing, but please avoid putting delete in the middle of the dropdown, as it is much easier to accidentally click it. As far as I know delete goes always last in our dropdowns, and most of the time has a separator before it. I would order them this way:

```
Install
Edit clone – I'm still not convinced by the name, but that's another topic ;)
---
Delete
```